### DISCUSSION TITLE
showing error

### DISCUSSION THREAD

**Discussion by friktiny** [Androz2091/AtlantaBot#491] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/76273378/120593766-aa1e9400-c45f-11eb-8cd3-74020fada500.png)
[
![image](https://user-images.githubusercontent.com/76273378/120593780-b145a200-c45f-11eb-9d40-4b1e666823ad.png)
](url) 
showing this error

**Comment by Androz2091** [comment, marked as answer]
You need to upgrade Node.js to at least v14.

**Reply by friktiny** [reply]
how to update?

**Reply by Zoriot** [reply]
https://phoenixnap.com/kb/update-node-js-version @friktiny
